<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cartDialog</class>
 <widget class="QDialog" name="cartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>612</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>463</width>
    <height>612</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>612</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>购物车</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelCount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>共计：件商品。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总价：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonComfirm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
   </property>
   <property name="text">
    <string>确认订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
   </property>
   <property name="text">
    <string>继续购物</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>559</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
   </property>
   <property name="text">
    <string>清空购物车</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>应付：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSale">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEvent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>满减。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>15</y>
     <width>71</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
   </property>
   <property name="text">
    <string>购物车</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/Icon/Icon/cart-white-48.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>tableView</zorder>
  <zorder>labelCount</zorder>
  <zorder>label</zorder>
  <zorder>labelTotal</zorder>
  <zorder>buttonComfirm</zorder>
  <zorder>buttonCancel</zorder>
  <zorder>buttonClear</zorder>
  <zorder>label_2</zorder>
  <zorder>labelSale</zorder>
  <zorder>labelEvent</zorder>
  <zorder>label_10</zorder>
  <zorder>label_4</zorder>
 </widget>
 <tabstops>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonComfirm</tabstop>
  <tabstop>buttonClear</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
